src/pages/NoLifespanToken/NoLifespanToken.module.scss

```diff
@@ -0,0 +1,27 @@
+.pageContainer {
+  height: 100vh;
```

Comment:
min-height? What if page content will be more than 100vh?

Reply:
no. this is a empty page always